spacelift/resource_gitlab_integration_test.go

```diff
@@ -72,7 +74,7 @@ func TestGitLabIntegrationResource(t *testing.T) {
 
 		testSteps(t, []resource.TestStep{
 			{
-				Config: configGitLab(host, token, descr, "null", "null"),
+				Config: configGitLab(host, token, descr, "null", "null", false),
```

Comment:
Nit: If we defined a const `useGitCheckout` at the top of the tests and used it here like `!useGitCheckout`, it would make what that bool means way more readable.